test/e2e/cordon-only-test

```diff
@@ -52,6 +52,7 @@ aemm_helm_args=(
   "$AEMM_DL_URL"
   --namespace default
   --set servicePort="$IMDS_PORT"
+  --set 'rbac.pspEnabled=true'
```

Comment:
why do we need to set this to true in all the tests now?

Reply:
not needed, I just needed to update the cluster psp since k8s 1.19 seems to default to seccomp to `runtime/default` rather than `docker/default`